tools/ReleaseNotes/data/whats-new-95.md

```diff
@@ -1281,6 +1711,26 @@ var scheduledJob = builder.AddProject<Projects.ScheduledWorker>("scheduled-worke
 
 This feature addresses issue #4366 and provides a unified development and deployment experience for both long-running services (Container Apps) and finite workloads (Container App Jobs) within your Aspire applications.
 
+#### Simplified job configuration overloads
+
+For common scenarios, Aspire 9.5 provides simplified overloads that reduce boilerplate code:
```

Comment:
I'm not sure this title and intro sentence make as much sense.

Reply:
embrace the slow. We will assign you to fix it.